vars/mavenCanaryRelease.groovy

```diff
@@ -9,12 +9,15 @@ def call(body) {
 
     def flow = new io.fabric8.Fabric8Commands()
     def utils = new io.fabric8.Utils()
+    def fmpPropertyName = "fabric8-maven-plugin.version"
+    def fmpPropertyValue = "3.5.38"
```

Comment:
@sthaha @rupalibehera @jaseemabid how can we make it parameterized instead of hardcoding it?

Reply:
@hrishin 
https://github.com/fabric8io/fabric8-pipeline-library/blob/master/vars/mavenCanaryRelease.groovy#L15 
and then access it this way,
or if you set something like this https://github.com/fabric8io/fabric8-pipeline-library/blob/master/vars/approve.groovy#L10 like a default value to some version so that you don't break any existing thing and you can use different version of fmp by just passing something like below:

In https://github.com/fabric8io/fabric8-pipeline-library/blob/master/vars/mavenCanaryRelease.groovy, you can do `def fmpPropertyValue= config.fmpPropertyValue ?: 3.5.38`
and then in Jenkisfile of a project which uses this do the below
Pass the value from Jenkinsfile 
```
        mavenCanaryRelease {
         fmpPropertyValue = '3.6'
        }
```
Then your project uses 3.6 passed via Jenkinsfile

Reply:
@hrishin , once you do the above let's see if the change works fine and then merge it.

Reply:
@rupalibehera that's definatelky better approach. I was wondering if we can keep these properties in pom.xml and release workflow of fmp could update the right version